build.sbt

```diff
@@ -270,9 +284,10 @@ lazy val mtagsShared = project
       "org.lz4" % "lz4-java" % "1.8.0",
       "com.google.protobuf" % "protobuf-java" % "3.25.1",
       "io.get-coursier" % "interface" % V.coursierInterfaces,
+      "com.lihaoyi" %% "requests" % V.requests,
```

Comment:
This will be a problem and has to be changed. We can't add any Scala dependencies directly to the compiler. This can be easily changed tho, so remote report context is implemented in metals, and passed to the PC via implicit. We don't need to implement it directly in the PC